### DISCUSSION TITLE
Onboarding PR is not showing up

### DISCUSSION THREAD

**Discussion by Mulugruntz** [mend/renovate-ce-ee#554] [Q&A]
Hi,

I'm trying to use Renovate CE, on GitHub (not on-premise).
I installed the app (from there https://github.com/apps/renovate ), clicked next and accept whenever prompted. And Now I see that:
![image](https://github.com/user-attachments/assets/88a68398-0dc0-49ea-bd6f-3f145f8f3df4)

---

And at the bottom of the page, I see the single repo that I've selected:
![image](https://github.com/user-attachments/assets/797fbfbb-ddbd-437f-886f-077964d5cab3)

And yet, after 14 hours, I don't see any activity by Renovate. Did I miss anything?

**Comment by rarkins** [comment]
Wrong repo sorry. Please raise a question in the main Renovate repo
